README.md

```diff
@@ -91,7 +92,7 @@ New members are advised to:
 - Submit a PR to add yourself to the [members list](#members).
 - Here are various ways to get involved:
   - Share your thoughts by joining the meetings, posting to the mailing list, discussions forum or Slack channel.
-  - Add a topic you would like to discuss to [the agenda](docs/meetings.md) of upcoming meeting.
+  - Add a topic you would like to discuss to [the agenda](docs/) of upcoming meeting.
```

Comment:
Thank you! 
Since you're at it, a good link here would be https://hackmd.io/xjK5ujQbTHSaEZjoY28b8g 
We take meeting notes in hackmd and sync them back to GitHub

Reply:
done

Reply:
Thank you!